src/views/VideoView/VideoView.tsx

```diff
@@ -21,17 +21,25 @@ import { GetVideo, GetVideoVariables } from '@/api/queries/__generated__/GetVide
 import { formatVideoViewsAndDate } from '@/utils/video'
 import { AddVideoView, AddVideoViewVariables } from '@/api/queries/__generated__/AddVideoView'
 import { ChannelLink } from '@/components'
+import { useInterval, usePersonalData } from '@/hooks'
 
 const VideoView: React.FC<RouteComponentProps> = () => {
   const { id } = useParams()
   const { loading, data, error } = useQuery<GetVideo, GetVideoVariables>(GET_VIDEO, {
     variables: { id },
   })
   const [addVideoView] = useMutation<AddVideoView, AddVideoViewVariables>(ADD_VIDEO_VIEW)
+  const { state, updateWatchedVideos } = usePersonalData()
 
+  const videoTimestamp = useMemo(() => {
+    const currentVideo = state.watchedVideos.find((v) => v.id === data?.video?.id)
+    return currentVideo?.__typename === 'INTERRUPTED' ? currentVideo.timestamp : null
+  }, [data?.video?.id, state.watchedVideos])
+
+  const videoRef = useRef<HTMLVideoElement>(null)
   const videoID = data?.video?.id
 
-  const [playing, setPlaying] = useState<boolean>(true)
+  const [playing, setPlaying] = useState<boolean>()
```

Comment:
Let's pass the default value explicitly

Reply:
That doesn't work because it causes the player to restart if you hit pause, I can pass in `false` as default though

Reply:
That's what I meant